tutorials/matchmaker101/matchmaker.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains the sample minimal pod definitions for all of the components that one may need to use Open Match as a match maker.
+# You can find the same pod definitions within the sub-folders under the /tutorials/ directory
+# Run `kubectl apply -f matchmaker.yaml` to deploy these definitions.
+
+apiVersion: v1
```

Comment:
From the tutorials' perspective, is it possible keep using minimal yaml definitions for the director, frontend, matchfunction, and evaluator? It looks a bit weird by using Pod and Deployment resources interchangeably in this yaml.

Reply:
Noted. I ll not block this one on the change given that it works and is transparent to actual tutorial steps but will update it to use service / pod in the next iteration.